internal/install/_static/terraform_deployer.yml

```diff
@@ -6,8 +6,8 @@ services:
     stop_grace_period: 5m
     environment:
       - TF_VAR_TEST_RUN_ID=${TF_VAR_TEST_RUN_ID:-detached}
-      - TF_VAR_REPO_NAME=${REPO_NAME:-unknown}
-      - TF_VAR_PULL_REQUEST=pr-${CHANGE_ID:-unknown}
-      - TF_VAR_CI_BUILD_NUMBER=${BUILD_NUMBER:-unknown}
+      - TF_VAR_CREATED_DATE=${CREATED_DATE:-unknown}
+      - TF_VAR_BRANCH_NAME=${BRANCH_NAME:-unknown}
+      - TF_VAR_BUILD_ID=${BUILD_ID:-unknown}
```

Comment:
Are you sure about BUILD_ID? The intention behind BUILD_NUMBER was to have a reference between PR and job run, for instance: PR-123-6, PR number is 123, 6th job run

Reply:
IIUC, both `build_id` and `build_number` are identical, as per https://beats-ci.elastic.co/pipeline-syntax/globals#env

> BUILD_NUMBER
The current build number, such as "153".
BUILD_ID
The current build ID, identical to BUILD_NUMBER for builds created in 1.597+, but a YYYY-MM-DD_hh-mm-ss timestamp for older builds.

Reply:
Keep in mind that GCP and AWS are picky about that tag format, so double-checking if these values are still fine.

Reply:
Good point, I just used the tag format in lowercase with underscores for AWS and GCP